### PR TITLE
Add solver-f34-py39 to CNV prod

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: thoth-station/s2i-thoth#184
Related: #663

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
/lgtm
thanks :100:

can we include this in here:
https://github.com/thoth-station/thoth-application/blob/master/solver/base/imagestreams.yaml
and other overlays as well.

File: solver/overlays/cnv-prod/imagestreamtag.yaml [hunk 1]
```diff
@@ -36,7 +36,35 @@ spec:
...
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: solver-fedora-34-py39
+spec:
+  tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: "quay.io/thoth-station/solver-fedora-34-py39:v1.9.0"
```
Comment (goern) [inline solver/overlays/cnv-prod/imagestreamtag.yaml:67]: this should be 1.10.1, right?

Comment (fridex) [reply solver/overlays/cnv-prod/imagestreamtag.yaml]: Adjusted in 24daa5cd

**Comment by fridex** [comment]
/unhold
/priority critical-urgent

If we want to increase service adoption, we need to have data for runtime environments we want to do recommendations on. CC @goern

**Pull request merged**